operations/helm/charts/mimir-distributed/templates/ingester/ingester-statefulset.yaml

```diff
@@ -15,6 +15,15 @@ metadata:
 spec:
   podManagementPolicy: {{ .Values.ingester.podManagementPolicy }}
   replicas: {{ $rolloutZone.replicas }}
+  {{- if and (semverCompare ">= 1.23-0" .Capabilities.KubeVersion.Version) (.Values.ingester.persistentVolume.enableStatefulSetAutoDeletePVC)  }}
```

Comment:
some people use the helm chart to first render the manifests and then apply them without using helm. In those cases the kubernetes version may not be accurate. Can you instead use the `mimir.kubeVersion` template that's in the chart?

Reply:
Let me know if looks better now in order to update the other sts commented above.